### PR TITLE
Add v1.5

### PR DESCRIPTION
**Pull request by aftersomemath**
Includes changes necessary for senior design sim, 1.5, and adding transforms back to the fc comms msp.

### CODE REVIEW

**Review by amiller27 — COMMENTED** [review]
Looks good pending other PR's getting merged

File: launch/passive_stack.launch [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <include if="$(eval str(arg('platform')) != '1.5')"
              file="$(find iarc7_sensors)/launch/obstacle_detector.launch" />
 
-    <include if="$(eval str(arg('platform')) != '1.5')"
+    <include if="$(eval str(arg('platform')) != '1.5' and str(arg('platform')) != 'sim_1.5')"
```
Comment (amiller27) [inline launch/passive_stack.launch]: What's the deal with this?  Why are there two versions of this param?

Comment (aftersomemath) [reply launch/passive_stack.launch]: One of them is for 1.5's sim and one of them is for the drone called 1.5.

**Review by amiller27 — APPROVED** [review]

File: launch/passive_stack.launch (hunk 1 above)
Comment (aftersomemath) [reply launch/passive_stack.launch]: I'm not a huge fan, eventually one of them will go away.

**Pull request merged**